app/routes/user/permissions.tsx

```diff
@@ -0,0 +1,300 @@
+            <input
+              type="hidden"
+              name="requestorSub"
+              value={endorsementRequest.requestorSub}
+            />
+            <Button
+              type="submit"
+              outline
+              disabled={disabled}
+              name="intent"
+              value="rejected"
+            >
+              Reject
+            </Button>
+          </rejectFetcher.Form>
+          <reportFetcher.Form method="post">
+            <input
+              type="hidden"
+              name="requestorSub"
+              value={endorsementRequest.requestorSub}
+            />
+            <Button
+              type="submit"
+              secondary
+              disabled={disabled}
+              name="intent"
+              value="reported"
+            >
+              Reject and Report
+            </Button>
+          </reportFetcher.Form>
+        </ButtonGroup>
+      </Grid>
+    )
+  } else {
+    return (
+      <Grid row style={disabled ? { opacity: '50%' } : undefined}>
+        <div className="tablet:grid-col flex-fill">
+          <div className="margin-y-0">
+            <strong>Requested Endorser: </strong>
+            {endorsementRequest.endorserEmail}
+          </div>
+          <div className="margin-y-0">
+            <strong>Status: </strong>
+            {endorsementRequest.status}
+          </div>
+        </div>
+        <ButtonGroup className="tablet:grid-col flex-auto flex-align-center">
+          <reportFetcher.Form method="post">
+            <input
+              type="hidden"
+              name="endorserSub"
+              value={endorsementRequest.endorserSub}
+            />
+            <Button
+              type="submit"
+              secondary
+              disabled={disabled}
+              name="intent"
+              value="delete"
+            >
+              Delete
+            </Button>
+          </reportFetcher.Form>
+        </ButtonGroup>
+      </Grid>
+    )
+  }
+}
+
+export function EndorsementRequestForm() {
+  const { validUsers } = useLoaderData<typeof loader>()
+  const [endorserSub, setEndorserSub] = useState('')
+
+  return (
+    <Form method="post">
+      <h2 id="modal-request-heading">Request Endorsement</h2>
+      <div className="usa-prose">
+        Requesting an endorsement from another user will share your email with
+        that individual. Please keep this in mind when submitting. Enter the
+        email of the individual you want to be endorsed by. They will receive a
+        notification alerting them to this request.
+      </div>
+      <Label htmlFor="endorserSub">Endorser</Label>
+      <input type="hidden" name="endorserSub" value={endorserSub} />
+      <ComboBox
+        id="endorserSubSelect"
+        name="endorserSubSelect"
+        options={validUsers}
+        onChange={(e) => setEndorserSub(e ?? '')}
+        defaultValue={''}
+      />
+      <Button
+        className="usa-button margin-top-2"
```

Comment:
The usa-button style is baked into this component, I think.

```suggestion
        className="margin-top-2"
```

Why is it necessary to control the margin?

Reply:
I'll remove that class, but I included the margin to give a bit of spacing. The way the buttons layout otherwise had it touching the bottom of the ComboBox and it looked a little too squished

Reply:
Please show screen shots.

Reply:
Without the margin specified:
![image](https://user-images.githubusercontent.com/11023698/208436266-80e13904-0859-4ba3-b5eb-fbb7f5c5b50a.png)

With:
![image](https://user-images.githubusercontent.com/11023698/208436327-8a90e88b-51d0-4401-8222-3842759be789.png)

Reply:
I think that's because you have it all wrapped in a `ButtonGroup`. Try removing the `ButtonGroup`.

Reply:
Nope, no button groups in that form, they are used in the cards